fix(cli): fix CLI flag of `bracketSpacing` option for lang-specific override (#5130)
biomejs/biome: .changeset/funny-beans-vanish.md

---
"@biomejs/biome": patch
---

> Fix a CLI flag `--bracket-spacing` is duplicated between the global configuration and the lang-specific override for JavaScript.

Reviewer: ```suggestion
Fixed the flag `--bracket-spacing` that was duplicated between the global configuration and the language-specific override for JavaScript.
```

Author: @arendjr Thank you for helping my bad English 😓
